website/versioned_docs/version-23.2/GettingStarted.md

```diff
@@ -91,6 +91,10 @@ yarn add --dev babel-jest babel-core regenerator-runtime
 > ```bash
 > yarn add --dev babel-jest babel-core@^7.0.0-bridge.0 @babel/core regenerator-runtime
 > ```
+>
+> You will need to use babel.config.js in order to transpile `node_modules`. See https://babeljs.io/docs/en/next/config-files for more information.
```

Comment:
Can we wrap `babel.config.js` with backticks for visibility?

Reply:
done!